dshowcapture.hpp

```diff
@@ -205,6 +205,11 @@ struct AudioConfig : Config {
 	AudioMode mode = AudioMode::Capture;
 };
 
+enum SettingsFlags {
+    Auto	= 0x1,
+    Manual	= 0x2
+};
```

Comment:
Where is this used?

Reply:
Before the json changes I had a settings structure that used it. Forgot it there. Already removed

Reply:
Awesome 👍